app/views/responses/index.html.erb

```diff
@@ -1,21 +1,59 @@
-<h2>Responses for this survey</h2>
-
-<table>
-  <thead>
-    <th>ID</th>
-    <th>Created at</th>
-    <th colspan="3"></th>
-  </thead>
-  <tbody>
+<div class="title-bar flex flex-v-center gutters">
+  <%= link_to root_path, title: 'Back to the dashboard', class: 'button--back flex flex-v-center flex-1 flex-h-start' do %>
+    <i class="material-icons">arrow_back</i><span>Dashboard</span>
+  <% end %>
+
+  <h1 class="h1--small-uppercase"><%= @survey.name %> results</h1>
+
+  <div class="title-bar--hidden"></div>
+</div>
+
+<div class="container--medium content-padding">
+
+  <div class="flex box--grey-light">
+    <div class="flex flex-1-half flex-column flex-v-center">
+      <p class="text-14"><%= @survey.user.organisation_name %></p>
+
+      <p class="text-40"><span class="bold"><%= @responses.count %></span> Result<% unless @responses.count == 1 %>s<% end %></p>
+
+      <%= link_to '#TODO', title: "Download the results for #{@survey.name}", class: 'button--download flex flex-v-center' do %>
+        CSV<i class="material-icons">file_download</i>  
+      <% end %> 
+    </div>
+
+    <div class="flex flex-1-half box--grey">
+      IMAGE?
+    </div>
```

Comment:
Just wanted to double check there is not meant to be an image here yet? Just in case!